examples/src/main/java/org/apache/ignite/examples/springdata/SpringDataExample.java

```diff
@@ -149,4 +153,30 @@ private static void queryRepository() {
         for (Long id: ids)
             System.out.println("   >>>   [id=" + id + "]");
     }
+
+    /**
+     * Execute queries with predicates over the repository.
+     */
+    private static void queryWithPredicates() {
+        System.out.println("\n>>> Persons with name 'John' or 'Brad':");
+
+        Iterable<Person> persons = repo.findAll(
+            QPerson.person.firstName.eq("John")
+                .or(
+                    QPerson.person.firstName.eq("Brad")
+                )
+        );
+
+        for (Person person: persons)
```

Comment:
Whitespace is expected before colon (:) also below.

Reply:
Done.